_data/retail.yml

```
@@ -301,18 +355,25 @@ websites:
     img: nordstrom.png
     twitter: Nordstrom
     facebook: Nordstrom
+    regions:
+      - us
```

```suggestion
    facebook: Nordstrom
```
Nordstrom [ships internationally](https://www.nordstrom.com/c/international-orders?jid=j009164-6899&cm_sp=merch-_-corp_6899_j009164-_-freelayout_corp_p99_details&;).